ush/get_mrms_files.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# -----------------------------------------------------------------------
+#
+# This function copies MRMS observation files from supported data
+# streams on disk to the specified directory. Files will be gathered for
+# a given time level, or minutes from a valid time.
+#
+# It's expected that this function will be called once for each time
+# level.
+#
+# A filelist of copied files will be prepared in the specified
+# directory.
+#
+# Global input variables from job
+#    PDY
+#    cyc
+#
+# Global input variables from var_defns
+#    RADARREFL_MINS
+#    NSSLMOSAIC
+#    OBS_SUFFIX
+#
+# -----------------------------------------------------------------------
+#
+
+function get_mrms_files () {
+
+  timelevel=$1
+  output_path=$2
+  mrms=$3
+
+  YYYY=${PDY:0:4}
+  MM=${PDY:4:2}
+  DD=${PDY:6:2}
+
+  echo "RADARREFL_MINS = ${RADARREFL_MINS[@]}"
+  NSSL=${NSSLMOSAIC}
+
+  # Link to the MRMS operational data
+  # This loop finds files closest to the given "timelevel"
+  for min in ${RADARREFL_MINS[@]}
+  do
+    min=$( printf %2.2i $((timelevel+min)) )
+    echo "Looking for data valid:"${YYYY}"-"${MM}"-"${DD}" "${cyc}":"${min}
```

Comment:
Maybe there is a reason behind all these double quotes, but you could just have one set of double quotes around this string.